scripts/generate.sh

```diff
@@ -51,6 +51,14 @@ helm template mysql bitnami/mysql \
   --set volumePermissions.enabled=true \
   > "${k8s_path}/mysql.yaml"
 
+echo "Packaging Manifests bitnami/postgres"
+helm template postgres bitnami/postgresql \
+  --set postgresqlUsername=user \
+  --set postgresqlPassword=password \
+  --set postgresqlDatabase=depscloud \
+> "${k8s_path}/postgres.yaml"
```

Comment:
I'm not sure on providing password as a param here. However, `.yaml` file that gets generated has a `secret` configuration and the StatefulSet is referring to the password in the secret and not this plain text `password`. Unable to connect the dots, so I thought I would call it out.

Reply:
Kubernetes secrets are Key-value maps where the value is base64 encoded.

Reply:
Documentation reference: https://kubernetes.io/docs/concepts/configuration/secret/